tsl/src/chunk_copy.c

```diff
@@ -547,33 +698,66 @@ chunk_copy_stage_delete_chunk(ChunkCopy *cc)
 }
 
 static const ChunkCopyStage chunk_copy_stages[] = {
+	/* Initial Marker */
+	{ CCS_INIT, NULL, chunk_copy_stage_init_cleanup },
```

Comment:
Curious if we want to have different stage here instead of init, basically one that inserts first activity record: `chunk_copy_activity_insert` to make it orthogonal with the cleanup function.

I think this would require to run this in separate transaction as well and complete current transaction in the `chunk_copy_init()` etc. This will allow us to remove the START also.

Another approach is to treat this as a special case and not a stage, since we update the activity table in init function, which is not a stage. This would simplify stuff a bit and the cleanup function logic as well